incubator/hnc/internal/reconcilers/object.go

```diff
@@ -659,8 +660,8 @@ func (r *ObjectReconciler) syncPropagation(ctx context.Context, log logr.Logger,
 // - Objects have a selector that doesn't match the destination namespace
 // - Service Account token secrets
 func (r *ObjectReconciler) shouldPropagateSource(log logr.Logger, inst *unstructured.Unstructured, dst string) bool {
-	selector := r.getSelector(log, inst)
-	treeSelector := r.getTreeSelector(log, inst)
```

Comment:
Did you actually delete these old functions?

Reply:
deleted now